pilot/pkg/config/kube/crd/conversion.go

```diff
@@ -102,6 +102,7 @@ func ConvertObject(schema collection.Schema, object IstioObject, domain string)
 	status, err := IstioStatusJSONFromMap(object.GetStatus())
 	if err != nil {
 		log.Errorf("could not get istio status from map %v, err %v", object.GetStatus(), err)
+		return nil, err
```

Comment:
I don't think we want to completely error out if there is an issue with status...cc @howardjohn

Reply:
OK，i've reverted it